.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: python
 env:
     - TOXENV=python
+    - DISTRO=lsb_release -si | tr '[:upper:]' '[:lower:]'
+    - CODENAME=lsb_release -sc
+    - PACKAGE=$(DISTRO).$(CODENAME).execline-dev_2.1.3.1-1_amd64.deb
+    - URL=https://github.com/bukzor/s6-packaging/releases/$(PACKAGE)
```

Comment:
nope.

This isn't bash syntax at all, and it belongs in the `CI install` script.